example/src/App.tsx

```diff
@@ -25,38 +30,71 @@ export type RootStackParamList = {
   Multiply: undefined
   HotspotBLE: undefined
   Account: undefined
+  CreateRandomHotspot: undefined
   AddGatewayTxn: undefined
   AssertLocation: undefined
   TransferHotspot: undefined
+  OraclePrice: undefined
+  HotspotList: undefined
 }
 
 export type RootNavigationProp = NativeStackNavigationProp<RootStackParamList>
 
 export default function App() {
+  const [address, setAddress] = useState('')
+  const [heliumWallet, setHeliumWallet] = useState('')
   LogBox.ignoreLogs([
     'Require cycle', // ignore HeliumJS require cycles
   ])
 
+  useEffect(() => {
+    const interval = setInterval(async () => {
+      try {
+        const nextAddr = await getAddressStr()
+        if (nextAddr === address) return
+
+        setAddress(nextAddr)
+        setHeliumWallet(nextAddr)
+      } catch {}
+    }, 2000)
+    return () => clearInterval(interval)
+  }, [address])
+
   return (
-    <OnboardingProvider baseUrl="https://onboarding.dewi.org/api/v2">
-      <HotspotBleProvider>
-        <NavigationContainer>
-          <Stack.Navigator
-            screenOptions={({ route }) => ({
-              presentation: 'modal',
-              headerShown: route.name === 'Home',
-            })}
-          >
-            <Stack.Screen name="Home" component={Home} />
-            <Stack.Screen name="Multiply" component={Mulitply} />
-            <Stack.Screen name="HotspotBLE" component={HotspotBLENav} />
-            <Stack.Screen name="Account" component={AccountNav} />
-            <Stack.Screen name="AddGatewayTxn" component={AddGatewayTxn} />
-            <Stack.Screen name="AssertLocation" component={AssertLocation} />
-            <Stack.Screen name="TransferHotspot" component={TransferHotspot} />
-          </Stack.Navigator>
-        </NavigationContainer>
-      </HotspotBleProvider>
-    </OnboardingProvider>
+    <SolanaProvider
+      cluster="devnet"
+      heliumWallet={heliumWallet}
+      solanaStatusOverride="complete"
+    >
+      <OnboardingProvider baseUrl="https://onboarding.oracle.test-helium.com/api">
```

Comment:
change `baseUrl` to `https://onboarding.dewi.org/api` if you want to test the real deal